src/components/Navbar/Navbar.jsx

```diff
@@ -53,13 +53,17 @@ const Navbar = () => {
       <Link to="/wishlist" className="btn-icon badge">
         <FaHandHoldingHeart className="btn-icon__icon" />
         <span className="btn-icon__text">Wishlist</span>
-        <span className="badge__count"> {wishlistItems?.length} </span>
+        {wishlistItems?.length > 0 && (
+          <span className="badge__count"> {wishlistItems?.length} </span>
+        )}
       </Link>
 
       <Link to="/cart" className="btn-icon badge">
         <AiOutlineShoppingCart className="btn-icon__icon" />
         <span className="btn-icon__text">Cart</span>
-        <span className="badge__count"> {cartItems.length} </span>
+        {cartItems?.length > 0 && (
```

Comment:
? : instead  of && may it may be giving error on console

Reply:
fixed - [e4f6c13](https://github.com/SukanyaSen530/WeebO-frontend/pull/11/commits/e4f6c1359a136c3468a711cafd7c72212e8396e0)